pkg/core/resources/apis/imports.go

```diff
@@ -0,0 +1,32 @@
+// Generated by tools/policy-gen
+// Run "make generate" to update this file.
+
+package policies
+
+type PluginInitializer struct {
+	InitFn      func()
+	Initialized bool
+}
+
+var NameToModule = map[string]*PluginInitializer{}
+
+func InitAllPolicies() {
+	for _, initializer := range NameToModule {
+		if !initializer.Initialized {
+			initializer.InitFn()
+			initializer.Initialized = true
+		}
+	}
+}
+
+func InitPolicies(enabledPluginPolicies []string) {
+	for _, policy := range enabledPluginPolicies {
+		initializer, ok := NameToModule[policy]
+		if ok && !initializer.Initialized {
+			initializer.InitFn()
+			initializer.Initialized = true
+		} else {
+			panic("policy " + policy + " not found")
+		}
+	}
+}
```

Comment:
Really seems to me that this code doesn't need to be generated (only NameToModule is dynamic here)